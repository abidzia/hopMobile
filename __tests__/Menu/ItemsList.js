import "react-native";
import React from "react";

// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

import ItemsList from "../../src/components/Menu/ItemsList";

it("renders ItemList correctly", () => {
  const tree = renderer.create(<ItemsList />);
  expect(tree).toMatchSnapshot();
});
