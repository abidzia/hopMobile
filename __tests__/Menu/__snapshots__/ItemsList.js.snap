// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ItemList correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 20,
        "textAlign": "center",
      }
    }
  >
    Select a section to view its items.
  </Text>
</View>
`;
